clientlib/go.mod

```diff
@@ -2,4 +2,9 @@ module github.com/etf1/kafka-message-scheduler/clientlib
 
 go 1.15
 
-require github.com/confluentinc/confluent-kafka-go v1.5.2
+require (
+	github.com/confluentinc/confluent-kafka-go v1.5.2
+	github.com/etf1/kafka-message-scheduler v0.0.0
```

Comment:
I think I don't have the choice because clientlib has its own go module and I cannot simply import `github.com/etf1/kafka-message-scheduler/retry` that is located in the parent module without requiring it in `go.mod`.

Maybe the `retry` package could be located into `clientlib` go module directly, what do you think?

Reply:
i didn't notice that and yes it should be in clientlib, all the files of this PR should be in clientlib